frontend/src/components/TemplatesIndex/TemplatesIndex.tsx

```diff
@@ -55,15 +70,64 @@ export const TemplatesIndex: FC<TemplateIndexProps> = () => {
             }
             return new Date(date).toLocaleDateString()
 
+        },
+        'lastUpdated': (d) => {
+            const date = Date.parse(d)
+            if (isNaN(date)) {
+                return new Date().toLocaleDateString()
+            }
+            return new Date(date).toLocaleDateString()
+
         }
     }
 
     const templateColumns = [
-        'name', 'updatedAt', 'createdBy', 'labName', 'status'
+        'name', 'lastUpdated', 'createdBy', 'lab', 'status', 'updatedAt', 'labName'
```

Comment:
This adds an additional column, we should rip through the localStorage version and rename `lastUpdated` to `updatedAt` instead of doing both. 

I assume the re-ordering of the columns is to match new designs?

Reply:
no this is support the old templates. there is another ticket to match new designs

Reply:
Then yeah, I would run the old templates through something like `templates.map(t => t.updatedAt = t.updatedAt || t.lastUpdated)`  before this. It's a hack but a temporary one

Reply:
setTemplates(JSON.parse(templatesJSON).map(template => ({ ...template, updatedAt: template.lastUpdated })))
i ended up using this